### ISSUE TITLE
Circle elements from Material Design icons are stripped out

### ISSUE THREAD

**Issue by jxnblk** [jxnblk/microicon#3]
The circle element attributes can be converted to path commands on build

**Comment by eli-oat** [comment]
Is this what leads to some of the icons missing detail, e.g. trello and youtube? 

![screen shot 2016-11-25 at 10 56 59 am](https://cloud.githubusercontent.com/assets/3342274/20630278/03319fec-b2fe-11e6-9e0d-136a8a1d751b.png)
![screen shot 2016-11-25 at 10 57 32 am](https://cloud.githubusercontent.com/assets/3342274/20630280/033d42e8-b2fe-11e6-97cb-5cf4fceb2fdc.png)

**Comment by peduarte** [comment]
@eli-oat yep, also [speakerdeck](http://icon.now.sh/speakerdeck), [styleshare](http://icon.now.sh/styleshare), [travisci](http://icon.now.sh/travisci), [windows](http://icon.now.sh/windows), [slides](http://icon.now.sh/slides), and probably more
